Sources/MySQL/Bind+Node.swift

```diff
@@ -133,6 +142,23 @@ extension UInt32 {
     }
 }
 
+extension UInt {
```

Comment:
Instead of duplicating the above method, we should cast to `UInt32` or use a protocol for all ints.